src/openfermion/utils/_remove_symmetry_qubits.py

```diff
@@ -0,0 +1,118 @@
+""" Module to remove two qubits from the problem space
+    using conservation of electron number and conservation
+    of electron spin. As described in arXiv:1701.08213.
+"""
+
+
+import openfermion
```

Comment:
if possible, it is best if you just import everything you need explicitly, as you do below. This way, you shouldn't need to "import openfermion".